### PR TITLE
Feature/np 1461 header links

### PR DESCRIPTION
**Pull request by davidsauntson**
Extend the header haml to allow a `current_site` bool, that renders a span instead of a link in the header links section.

### CODE REVIEW

File: scss/6-components/_navigation_greedy-nav.scss [hunk 1]
```diff
@@ -51,6 +52,10 @@
       }
     }
 
+    span {
```
Comment (davidsauntson) [inline scss/6-components/_navigation_greedy-nav.scss:55]: a conscious choice so as not to have styling classes in the GreedyNav typescript

File: scss/6-components/_search.scss [hunk 2]
```diff
@@ -75,3 +75,18 @@
     }
   }
 }
+
+/* stylelint-disable plugin/selector-bem-pattern */
+.cads-header__search-row {
```
Comment (davidrapson) [inline scss/6-components/_search.scss:80]: If possible we should avoid touching other component classes, could we add a `cads-header__search-form` wrapper so we could write:

```
.cads-header__search-form {
    margin-left: auto;

    @include cads-media-breakpoint-only(md) {
      width: 70%;
    }

    @include cads-media-breakpoint-only(lg) {
      width: 55%;
    }
}
```

Avoids the need for the `stylelint-disable` then

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,7 @@
 **New**
 
 - 📋 Footer Feedback: add a class to allow selection of footer feedback link
+- 🔗 Header: Links in the header now accept a `current_value` hash property to render a span instead of a link
```
Comment (davidrapson) [inline CHANGELOG.md]: Should `current_value` be `current_site`?

**Review by davidrapson — APPROVED** [review]
👍 Looks good, with a docs change and a question about BEM styles for the search form

File: scss/6-components/_search.scss (hunk 2 above)
Comment (davidsauntson) [reply scss/6-components/_search.scss:80]: well that's a good idea 👍

**Comment by davidrapson** [comment]
🚀

**Pull request merged**